pkg/codegen/go/gen_program_utils.go

```diff
@@ -70,18 +70,18 @@ func getHelperMethodIfNeeded(functionName string, indent string) (string, bool)
 				return pulumi.String(string(data))
 			}`, true
 	case "filebase64":
-		return `func filebase64OrPanic(path string) pulumi.StringPtrInput {
+		return `func filebase64OrPanic(path string) string {
```

Comment:
It is important that these functions generate code with a type signature equal to that specified during PCL binding. For `filebase64 : string -> string` the generated code should return plain values rather than `pulumi.StringPtrInput` and it is the responsibility of program-gen to _lift_ that into `Input[T]` from `T` when required. (although that functionality is currently very clunky)